test/studies/twitter/sleepsort.chpl

```diff
@@ -0,0 +1,11 @@
+use Time;
```

Comment:
Could be.  I'll move to a smaller set of more distantly spaced integers if it does cause problems.  I'm also wondering whether we'll have any portability problems with the file.length logic in the check140*.chpl tests or generally with survey.chpl.